README.md

```diff
@@ -1,2 +1,18 @@
-# pygenesys
-Python for Generating Energy Systems
+# Python for Generating Energy Systems (PyGenESys)
+
+This package can be used to generate input files for the [Temoa](https://github.com/temoaproject/temoa) modelling tool. Transparency and repeatability are integral to the design motivations for Temoa. PyGenesys takes this concept
```

Comment:
U.S. spelling is usually "modeling" but "modelling" is used everywhere else, just figured I'd point it out